Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -683,6 +683,52 @@ class OpenFiftyTabsDebugOption: HiddenSetting {
     }
 }
 
+class UseNewHistoryApiOption: HiddenSetting {
+    let profile: Profile
+
+    override init(settings: SettingsTableViewController) {
+        self.profile = settings.profile
+        super.init(settings: settings)
+    }
+
+    override var title: NSAttributedString? { switch self.profile.historyApiConfiguration {
```

Comment:
nit, remove self usage when not necessary
```suggestion
    override var title: NSAttributedString? { switch profile.historyApiConfiguration {
```